src/explore-education-statistics-common/src/modules/charts/components/MapBlockInternal.tsx

```diff
@@ -575,7 +604,7 @@ export const MapBlockInternal = ({
             <FormSelect
               name="selectedLocation"
               id={`${id}-selectedLocation`}
-              label="2. Select a location"
+              label={`2. Select a ${locationType}`}
```

Comment:
We should probably switch between "a" and "an" depending on whether or not the locationType starts with a vowel or not